### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by KeisukeShima**
Add description for each hook.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,3 +16,27 @@ repos:
...
+
+Apply prettier to launch.xml
+
+### `prettier-package-xml`
+
+Apply prettier to package.xml
+### `sort-package-xml`
+
+Sorts the names contained in each tag in package.xml.
+If you want to exclude a tag from the sort, add `<! -- nolint -->` at the beginning of the line.
```
Comment (kenji-miyake) [inline README.md:36]: I'm not so confident, but I feel this sounds better.

```suggestion
If you want to exclude a tag from sorting, add `<! -- nolint -->` at the beginning of the line.
```

File: README.md [hunk 2]
```diff
@@ -16,3 +16,27 @@ repos:
...
+
+Example
```
Comment (kenji-miyake) [inline README.md:38]: [IMO] This is redundant.

```suggestion
```

Comment (kenji-miyake) [reply README.md:38]: Or, writing like this (`, for example:`) looks good.

https://github.com/pre-commit/pre-commit-hooks#sort-simple-yaml

File: README.md [hunk 3]
```diff
@@ -16,3 +16,27 @@ repos:
       - id: prettier-package-xml
       - id: sort-package-xml
 ```
+
+## Hooks available
+
+### `prettier-xacro-xml`
```
[the remark this answers is not in the document]
Comment (KeisukeShima) [reply README.md]: OK. a6cfc03

[the remark this answers is not in the document]
Comment (kenji-miyake) [reply README.md]: Ah, it was from here.
https://github.com/pre-commit/pre-commit-hooks#hooks-available

File: README.md [hunk 4]
```diff
@@ -16,3 +16,25 @@ repos:
...
+Apply prettier to [xacro](http://wiki.ros.org/xacro).
+
+### prettier-launch-xml
+
+Apply prettier to [launch.xml](https://design.ros2.org/articles/roslaunch_xml.html).
+
+### prettier-package-xml
+
+Apply prettier to [package.xml](https://www.ros.org/reps/rep-0149.html).
+### sort-package-xml
```
Comment (kenji-miyake) [inline README.md:33]:
```suggestion
Apply prettier to [package.xml](https://www.ros.org/reps/rep-0149.html).

### sort-package-xml
```

File: README.md [hunk 5]
```diff
@@ -16,3 +16,26 @@ repos:
...
+
+Apply prettier to [package.xml](https://www.ros.org/reps/rep-0149.html).
+
+### sort-package-xml
+
+Sort the dependent packages in [package.xml](https://www.ros.org/reps/rep-0149.html).
+If you want to exclude a tag from sorting, add `<! -- nolint -->` at the beginning of the line.
+
+```xml
+<!-- nolint --> <depend>rclcpp</depend>
```
Comment (kenji-miyake) [inline README.md:40]: [Nits] It's not `lint`.

```suggestion
<!-- no format --> <depend>rclcpp</depend>
```

File: README.md [hunk 6]
```diff
@@ -16,3 +16,26 @@ repos:
...
+Apply prettier to [launch.xml](https://design.ros2.org/articles/roslaunch_xml.html).
+
+### prettier-package-xml
+
+Apply prettier to [package.xml](https://www.ros.org/reps/rep-0149.html).
+
+### sort-package-xml
+
+Sort the dependent packages in [package.xml](https://www.ros.org/reps/rep-0149.html).
+If you want to exclude a tag from sorting, add `<! -- nolint -->` at the beginning of the line.
```
Comment (kenji-miyake) [inline README.md:37]:
```suggestion
If you want to exclude a tag from sorting, add `<! -- no format -->` at the beginning of the line.
```

File: README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (KeisukeShima) [reply README.md]: Yes. But it's not necessary.

**Review by kenji-miyake — APPROVED** [review]
LGTM!

**Pull request merged**